dev/io.openliberty.faces.4.0.internal/src/io/openliberty/faces40/internal/ee/WASMyFacesContainerInitializer.java

```diff
@@ -83,6 +84,23 @@ public void onStartup(Set<Class<?>> clazzes, ServletContext servletContext) thro
             addLifecycleListener(servletContext);
 
             log.log(Level.INFO, "Added StartupServletContextListener to the servlet context");
+
+            /*
+             * In previous version of Faces/JSF this was done in the AbstractJSPExtensionFactory.createExtensionProcessor.
+             * In Faces 4.0 the Pages/JSP feature is no longer enabled as Pages/JSP support was removed from the Faces 4.0 Specification.
+             * If an AnnotationHandlerManager is not added here then when the WASCDIAnnotationInjectionProvider tries to get an instance
+             * of an AnnotationHelperManager null is returned.
+             *
+             * The JSFExtensionFactory will do this for applications that define a FacesServlet and we'll create the
```

Comment:
Shouldn't the comment say WASMyFacesContainerInitializer instead?

Reply:
Nope, the comment is correct, each comment talks about the other class so it is clear where the logic is taking place.